### PR TITLE
Markets in suspension can be resumed or terminated after a protocol u…

### PR DESCRIPTION
**Pull request by vega-paul**
…pgrade restarts the network

### CODE REVIEW

**Review by gordsport — CHANGES_REQUESTED** [review]
Please also add the new AC to:

- https://github.com/vegaprotocol/specs/blob/cosmicelevator/protocol/features.json

BEFORE it is merged - if applicable to a cosmic elevator feature

File: protocol/0028-GOVE-governance.md [hunk 1]
```diff
@@ -531,6 +531,7 @@ APIs should also exist for clients to:
 - Enacting a market closure governance proposal on a market that is not in a settled state always uses the price supplied with the proposal for final settlement, even when the oracle settlement price is available at that time. (<a name="0028-GOVE-138" href="#0028-GOVE-138">0028-GOVE-138</a>)
 - Successful enactment of a market closure proposal changes the state of the market to `closed`. (<a name="0028-GOVE-139" href="#0028-GOVE-139">0028-GOVE-139</a>)
 - Attempt to enact a market closure proposal on a closed market has no effect. (<a name="0028-GOVE-111" href="#0028-GOVE-111">0028-GOVE-111</a>)
+- Markets in suspension can be resumed or terminated after a protocol upgrade restarts the network. (<a name="0028-GOVE-145" href="#0028-GOVE-145">0028-GOVE-145</a>)
```
Comment (Jiajia-Cui) [inline protocol/0028-GOVE-governance.md]: Markets is suspended by governance
 
 can be split into 2 ACs

**Review by Jiajia-Cui — DISMISSED** [review]
LGTM

**Comment by vega-paul** [comment]
@gordsport https://github.com/vegaprotocol/specs/pull/1868

**Review by gordsport — APPROVED** [review]
Approving as the content has not changed since the last approval

**Pull request merged**